### PR TITLE
Add pending item information

### PR DESCRIPTION
**Pull request by lee-dohm**
Since we're getting questions about pending items and how to confirm them, I wanted to get this section into the manual.

### CODE REVIEW

File: book/02-using-atom/sections/03-interface.asc [hunk 1]
```diff
@@ -24,6 +24,20 @@ Each pane has its own ``items'' or files, which are represented by tabs. You can
 
 To close a pane, close all its editors with `cmd-w`, then press `cmd-w` one more time to close the pane. You can configure panes to auto-close when empty in the Settings view.
 
+[[_pending_files]]
+=== Pending Files
+
+When you open a new file by single-clicking in the tree view, it will open in a new tab with an italic title. This indicates that the file is "pending". When a file is pending, it will be replaced by the next pending file that is opened. This allows you to click around the tree view to find that file you know is there but you don't remember the exact name of without opening a dozen tabs and then having to go back and close them all.
```
Comment (mnquintana) [inline book/02-using-atom/sections/03-interface.asc]: The last sentence here is quite a sentence :stuck_out_tongue_closed_eyes: – maybe it's worth breaking this out into a more formal example?

**Comment by mnquintana** [comment]
I think it's worth mentioning somewhere that pending items used to be known as preview tabs

**Comment by thedaniel** [comment]
@lee-dohm are you happy with this now, or do you want to hold it in case there are any chance the default changes in beta?

**Comment by lee-dohm** [comment]
Thanks for the reminder ... I'll update it to read "Pending Pane Items" since the name changed. Then I'll be happy with it :grinning:

**Comment by BinaryMuse** [comment]
@lee-dohm is this good to merge?

**Pull request merged**